src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -66,6 +67,8 @@ public String getAuthToken() {
         return authToken;
     }
 
+    public String getCredentialsAuthTokenId() { return credentialsAuthTokenId; }
```

Comment:
For consistency with the other methods, please break this into 4 lines, i.e. `return ...` on its own line, `{` and `}` on their own lines